shared/common-adapters/section-list.desktop.tsx

```diff
@@ -192,6 +192,7 @@ class SectionList<T extends Section<any, any>> extends React.Component<Props<T>,
   }
 
   _flatten = memoize((sections: ReadonlyArray<T>) => {
+    this._sectionIndexToFlatIndex = []
```

Comment:
This was a bug (my bad) that didn't affect us until this PR where we start to mutate section list data after it's mounted.